.github/workflows/functional.yml

```diff
@@ -0,0 +1,76 @@
+name: Functional Tests
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+env:
+   TEST_IMAGE_NAME: fiware/orion-ci:rpm7
+
+jobs:
+  functional-1:
+    runs-on: ubuntu-18.04
+
+    services:
+      mongodb:
+        image: mongo:4.0
+        ports:
+          - 27017:27017
```

Comment:
```suggestion
    # FIXME: currently mongodb runs inside fiware/orion-ci:rpm7 image. Once it gets removed
    # from it, this part will be enabled
    #services:
    #  mongodb:
    #    image: mongo:4.0
    #    ports:
    #      - 27017:27017
```

Reply:
(I'm doing the suggestion only in this .yml but it should be applied also to other .yml files where it makes sense)